BussinesObject/Api/Steps/ApiUserSteps.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Bogus;
+using BussinesObject.Api.RestEntities;
+using BussinesObject.Api.Services;
+using Newtonsoft.Json;
+using NUnit.Allure.Attributes;
+using NUnit.Framework;
+using System.Net;
+
+namespace Api.BusinessObject.Steps
+{
+    public class ApiUserSteps : UserService
+    {
+        public ApiUserSteps() : base()
+        {
+        }
+
+        [AllureStep]
+        public User CreateRandomUser() 
+        {
+            Faker faker = new Faker();
+
+            var userModel = new CreateUserModel()
+            {
+                FirstName = $"{faker.Person.FirstName}",
+                LastName = $"{faker.Person.LastName}",
+                Email = $"{faker.Person.Email}",
+                Password = "P@ssw0rd!",
+                PasswordConfirmation = "P@ssw0rd!",
+                Admin = false
```

Comment:
store password in config or use faker if it's random password

Reply:
get password from faker